content/en/docs/contribute/start.md

```diff
@@ -294,15 +294,48 @@ contribution guide.
 
 ## Review docs pull requests
 
-People who are not yet approvers or reviewers can still review pull requests. 
-The reviews are not considered "binding", which means that your review alone 
-won't cause a pull request to be merged. However, it can still be helpful. Even 
-if you don't leave any review comments, you can get a sense of pull request 
-conventions and etiquette and get used to the workflow. Take a look at the 
-[Content](/docs/contribute/style/content-guide/) and 
-[Style](/docs/contribute/style/style-guide/) guides before you review so you 
+People who are not yet approvers or reviewers can still review pull requests. You
+learn the code base, and you begin to build trust with your fellow contributors.
+You don't need to be a good developer or writer to review pull requests, but you do need
```

Comment:
Remove this sentence.
```suggestion
```

Reply:
I see your point, but I didn't infer that omitting opinion is part of _Use simple and direct language_. I'd like to emphasize that people who think of themselves as newbies can still review PRs, and I'm not convinced that "People who are not yet approvers or reviewers can still review pull requests" quite covers that.  How about "People who are new to the Docs can still review pull requests." ?

Reply:
How about:
> People who are new to documentation can still review pull requests.